src/components/screen/__tests__/Chat.test.tsx

```diff
@@ -68,6 +71,8 @@ describe('[Login] interaction', () => {
   it('should call [setShowMenu] when focused', () => {
     const touchMenu = testingLib.getByTestId('touch_menu');
     fireEvent.press(touchMenu);
+
+    expect(setShowMenu).toHaveBeenCalledTimes(1);
```

Comment:
This test was failed. What's wrong? @hyochan

Reply:
As told previously, `toHaveBeenCalledTimes` works when you've mocked the function with `jest`. This is matter with checking state instance and `react-testing-library` generally don't deal with the instance with it's own philosophy ([reference link](https://github.com/testing-library/react-testing-library#guiding-principles)). If you've not quite got what this is about, it is pretty simple and I can make you more clear in `offline` :relaxed:

P.S. If you want to continue with the above testing, you should check the `dom` changes. Which means, you can check which `elements` to render when the `showMenu` is false and when it is true!

Reply:
So, I mocked `setShowMenu`! but I don't understand exactly. 
I'll listen to the explanation offline at the next meetup! :)